schema.graphql

```diff
@@ -1,3 +1,5 @@
+extend schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@shareable"])
+
```

Comment:
I'm assuming we need this to use `@constraint` directive?

Reply:
It was in the example documentation, so I was wondering if it was a fed 2 thing... but no, I don't think so... I'm still getting composition errors. I'll futz around with the schema tomorrow and see if I can get it to compose.

Reply:
I think actually I have to follow this: https://www.apollographql.com/docs/federation/federated-types/federated-directives/#managing-custom-directives